Restructure docs menu

tmhastings opened the pull request:
Reorganize the Docs Menu (index.md)

Add a "New User Guide"

Changes can be viewed at this link: https://oiaps-tmhastings.readthedocs.io/en/restructure-docs-menu/

tmhastings committed:
Merge pull request #3 from nightscout/dev

Update tmhastings branch with nightscout repo dev branch

tmhastings committed:
Create New User Setup

Add a New User Setup Guide

tmhastings committed:
Create close-loop-video

tmhastings committed:
Add files via upload

tmhastings committed:
Delete docs/EN/Setup/img/close-loop-video

tmhastings committed:
Rename New User Setup to New-User-Setup.md

tmhastings committed:
Update New-User-Setup.md

Create a New User Setup Guide

tmhastings committed:
Update index.md

Reorganize the Menu

tmhastings committed:
Update index.md

Correct formatting issue

tmhastings committed:
Update New-User-Setup.md

Test link correction

tmhastings committed:
Update New-User-Setup.md

Correct formatting errors

tmhastings committed:
Add files via upload

tmhastings committed:
Update New-User-Setup.md

tmhastings committed:
Update New-User-Setup.md

tmhastings committed:
Update New-User-Setup.md

tmhastings committed:
Update index.md

tmhastings committed:
Rename Close Loop Video.mp4 to CloseLoopVideo.mp4

tmhastings committed:
Update New-User-Setup.md

tmhastings committed:
Delete docs/EN/Setup/img/CloseLoopVideo.mp4

Remove Video (to be replaced with image)

tmhastings committed:
Add files via upload

tmhastings committed:
Update New-User-Setup.md

Insert image where video was removed.

tmhastings committed:
Update index.md

- Reorganize the menu to have "Build" instructions higher in the menu.
-Redistribute components of the "Using Trio" menu into better-suited areas

tmhastings committed:
Update transition-qa.md

Change Heading format for "Introduction" and "I want to try trio..." to remove it from menu preview

tmhastings committed:
Update index.md

tmhastings committed:
Update smbsettings.md

Correct Heading state for "Are SMBs Allowed?"

tmhastings committed:
Update troubleshoot.md

tmhastings committed:
Update New-User-Setup.md

Test to fix link

tmhastings committed:
Update New-User-Setup.md

Another attempt to fix the link

tmhastings committed:
Update New-User-Setup.md

Another attempt to fix the link...

tmhastings committed:
Update New-User-Setup.md

tmhastings committed:
Update New-User-Setup.md

Maybe this will work?

tmhastings committed:
Update New-User-Setup.md

Just checking for Jonas... ;-)

tmhastings committed:
Update New-User-Setup.md

Andreas to the rescue!

MikePlante1 commented on docs/EN/Configuration/transition-qa.md:
```diff
@@ -145,7 +144,7 @@ Here's the recommendation:
 
 Are you using a commercial closed-loop system like Insulet's Omnipod 5, Tandem's Control IQ, Medtronic's 780G, etc.? If you're happy, stay if you're not getting the desired results; read through the chapter on "Coming from iOS Loop" to see the main issues that Trio solves.
 
-## I Want to Try Trio. How Do I Proceed?
```
I think this should remain its own subheading rather than being moved into `## Coming From Commercial Closed-Loop Systems`, so I would recommend reverting this line.

tmhastings replied:
It looks odd in the side menu when it's a ## Heading as well. I changed the formatting so it avoids both issues of falling under Commercial Closed loop systems and looking out of place in the side menu. (Commit #6b4db1e)

MikePlante1 commented on docs/EN/index.md:
```diff
@@ -51,96 +51,112 @@ You can also provide support in online support groups by helping users adjust th
+
+Temp Targets <./operate/temptarget.md>
+
+Customizations <./operate/customize.md>
```
I think `Customizations <./operate/customize.md>` fits better under `Build Instructions <./operate/build.md>` on line 84

tmhastings replied:
Agreed!

MikePlante1 commented on docs/EN/Setup/New-User-Setup.md:
```diff
@@ -0,0 +1,86 @@
+# New User Setup
+To set up your Trio app, touch the settings icon located on the bottom right of the main screen. Follow this step-by-step guide to start using Trio. Each step contains links to information on setting up each part of your Trio app. Use this as your homebase to reference back to as you set up your app.
```
How about adding a ⚙️ next to `settings icon` to make it extra clear?

MikePlante1 commented on docs/EN/Setup/New-User-Setup.md:
```diff
@@ -0,0 +1,86 @@
+To set up your Trio app, touch the settings icon located on the bottom right of the main screen. Follow this step-by-step guide to start using Trio. Each step contains links to information on setting up each part of your Trio app. Use this as your homebase to reference back to as you set up your app.
+<br>
+
+## Step 1: Connect your Devices
```
Should `Build Trio` be included as Step 1 and move the others down one? Or maybe that doesn't fit into this guide?

tmhastings replied:
This is a guide to set up the app after it's built. I can add something in the intro to the Guide that clarifies that.

MikePlante1 commented on docs/EN/index.md:
```diff
@@ -51,96 +51,112 @@ You can also provide support in online support groups by helping users adjust th
+
+CR <./settings/configuration/carbratios.md>
+
+ISF <./settings/configuration/insulinsensitivities.md>
```
Maybe move `ISF` above `CR` for ranking of importance, and also to match the order in [New User Setup](https://oiaps-tmhastings.readthedocs.io/en/restructure-docs-menu/Setup/New-User-Setup.html#step-2-add-profile-settings)?

MikePlante1 commented on docs/EN/Setup/New-User-Setup.md:
```diff
@@ -0,0 +1,86 @@
+:::{warning}
+<b>DO NOT ENABLE DYNAMIC SETTINGS YET</b>
+
+It is essential that Trio have enough data to make sound recommendations. This takes a minimum of 7-14 days. It is not advised to enable dynamic settings until BOTH criteria below are met:
```
Dynamic only uses the past 10 days of data, with a minimum of 1 day (technically 21 hours). I think the more important part about waiting to turn on dynamic isn't so much waiting for data (since the only data that's collected is TDD) but rather tuning the primary settings and getting comfortable with using the app/oref.

MikePlante1 commented on docs/EN/Setup/New-User-Setup.md:
```diff
@@ -0,0 +1,86 @@
+ 3. [Max COB](../settings/configuration/preferences/mainsettings.md#max-cob)
+ 4. [Bolus Increment](../settings/configuration/preferences/smbsettings.md#bolus-increment)
+ 5. [Enable SMB](../settings/configuration/preferences/smbsettings.md)
+ 6. [Max Delta-BG Threshold SMB](../settings/configuration/preferences/smbsettings.md#max-delta-bg-threshold-smb)
```
`Max Delta-BG Threshold` seems like more of an advanced setting to me. I think the default `0.2` is fine for new users.

tmhastings replied:
Removed. Only added because it was in the Setup guide on the website and that was my template. I think your point is valid and I agree.

MikePlante1 commented on docs/EN/index.md:
```diff
@@ -51,96 +51,112 @@ You can also provide support in online support groups by helping users adjust th
+
+ISF <./settings/configuration/insulinsensitivities.md>
+
+Set Your Target BG <./settings/configuration/targetglucose.md>
```
I'd prefer to keep this `Target Glucose` and remove the `Set Your` part since the others aren't `Set Your ISF`. Bit of a pet peeve that OS-AID systems use sgv rather than bg but glucose covers both.

MikePlante1 commented on docs/EN/index.md:
```diff
@@ -51,96 +51,112 @@ You can also provide support in online support groups by helping users adjust th
+
+Close the Loop <./Configuration/Configure.md>
+
+Enable SMBs <./settings/configuration/concepts/smb-uam.md>
```
I'd recommend just titling this `SMB and UAM`

MikePlante1 commented:
Great job on this @tmhastings! I added a few comments and confirmed all the links in the menu worked and that all of the links in the current menu are included in this restructured menu.

tmhastings committed:
Update transition-qa.md

tmhastings committed:
Update index.md

tmhastings committed:
Update New-User-Setup.md

Add ⚙️ icon
Resolves: https://github.com/nightscout/trio-docs/pull/48#discussion_r1660178307

tmhastings committed:
Update New-User-Setup.md

Add link to build instructions and clarify the purpose of the setup guide in the introduction.
Resolves: (https://github.com/nightscout/trio-docs/pull/48/files#r1660179493)

tmhastings committed:
Update New-User-Setup.md

Enact great points and ideas from @mikeplante1
Resolves: https://github.com/nightscout/trio-docs/pull/48/files#r1660182730

tmhastings committed:
Update index.md

Match order in New User Setup Guide
Resolves: https://github.com/nightscout/trio-docs/pull/48/files#r1660180754

tmhastings committed:
Update index.md

- Change Title from "Enable SMBs" to "SMB & UAM"
Resolves: https://github.com/nightscout/trio-docs/pull/48/files#r1660185925
- Reorder so "Close the Loop" is below all User Settings in the menu

tmhastings committed:
Update index.md

Relocate "Customizations" to under the Build Menu
Resolves: https://github.com/nightscout/trio-docs/pull/48/files#r1660178024

tmhastings reviewed (commented):
Resolved Issues. Please review, @MikePlante1

marionbarker reviewed (approved):
Changes look good. LGTM.
